plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/figures/GroupFigure.java

```diff
@@ -68,16 +70,32 @@ private void createMainFigure() {
 		mainFigure.setOpaque(false);
 
 		final GridLayout mainLayout = new GridLayout(1, true);
+		mainLayout.marginHeight = 0;
+		mainLayout.marginWidth = 0;
 		mainLayout.verticalSpacing = 0;
 		mainLayout.horizontalSpacing = 0;
 		mainFigure.setLayoutManager(mainLayout);
 	}
 
-	private void createCommentFigure() {
+	private IFigure createCommentFigure() {
 		commentFigure = new InstanceCommentFigure();
 		commentFigure.setCursor(Cursors.SIZEALL);
 		final AdvancedLineBorder commentBorder = new AdvancedLineBorder(PositionConstants.SOUTH);
 		commentFigure.setBorder(commentBorder);
+
+		final Figure commentContainer = new Figure();
+		commentContainer.setLayoutManager(new ToolbarLayout());
+
+		final int lineHeight = (int) CoordinateConverter.INSTANCE.getLineHeight();
```

Comment:
what is this doing ? 
Why dont you use top = Math.floor(lineHeight) ?

Reply:
This code is applying half of the line height above and half of it below the comment text. As we may have an uneven line height we need to compensate for rounding errors. Putting all above the comment would result in a strange look.